content/concepts/loaders.md

```diff
@@ -28,7 +28,7 @@ Secondly, configure in your `webpack.config.js` that for every `.css` file the `
 module.exports = {
   module: {
     rules: [
-      {test: /\.css$/, use: ['css-loader'](/loaders/css-loader)},
+      {test: /\.css$/, use: ['css-loader'](https://github.com/webpack/css-loader)},
```

Comment:
That was one of the links that was failing but I guess only locally for me, now that I go to the live site I see that page does exist -- weird, does do I need to run more than just `npm run build` to pick up all the remote content?